src/content/en/tools/chrome-devtools/debug/console/console-ui.markdown

```diff
@@ -48,6 +48,8 @@ To open the dedicated **Console** panel, either:
   <kbd>Cmd</kbd>+<kbd>Opt</kbd>+<kbd class="kbd">J</kbd> (Mac).
 * If DevTools is already open, press the **Console** button.
 
+When you open the Console panel, the Console drawer automatically closes. 
```

Comment:
Nit: "closes automatically" sounds a little more natural to me. I also would have used the word "collapse", as the thing is still there at the bottom.